warehouse/README.md

```diff
@@ -3,9 +3,42 @@
 This dbt project is intended to be the source of truth for the cal-itp-data-infra BigQuery warehouse.
 
 ##  Setting up the project on your local machine
-0. [Authorize the gcloud CLI](https://cloud.google.com/sdk/docs/authorizing) so that you can access BigQuery (implied step -1: make sure that you have permissions for BigQuery)
-1. Create a file `~/.dbt/profiles.yml` (i.e., in your home directory inside a `.dbt` subdirectory) that contains the following:
-```
+
+> Note: if you get `Operation not permitted` when attempting to use the terminal,
+> you may need to [fix your terminal permissions](https://osxdaily.com/2018/10/09/fix-operation-not-permitted-terminal-error-macos/)
+
+### Install Homebrew (if you haven't)
+1. Follow the installation instructions at https://brew.sh/
+2. Then, `brew install gdal` to install a geospatial library needed later
+
```

Comment:
should add a section here saying that people need to clone the `data-infra` repo if they haven't